ui/pages/settings/settings-search-list/settings-search-list.js

```diff
@@ -29,32 +29,32 @@ export default function SettingsSearchList({ results, onClickSetting }) {
                 />
 
                 <span
-                  id={`menu-tab_${id}`}
+                  id={`menu-tab_${route}`}
                   className={classnames(
                     'settings-page__header__search__list__item__tab',
                     {
                       'settings-page__header__search__list__item__tab-multiple-lines':
-                        tab === t('securityAndPrivacy'),
+                        tabKey === 'securityAndPrivacy',
                     },
                   )}
                 >
-                  {tab}
+                  {t(tabKey)}
```

Comment:
This is problematic, and the reason that the original implementation had passed the t method around. Setting variables to message keys and passing those around become a nightmare for figuring out how  messages are being used, and they result in our strict evaluation of where message keys are being used to fail.

Reply:
Cool, made some changes